README.md

```diff
@@ -25,23 +25,41 @@ A bit about this module
 | Name | Version |
```

Comment:
Update the readme to have the right checks and a description. Also add a note about the `Patch Group` tag that needs to be applied.